cmd/cosign/cli/upload.go

```diff
@@ -96,7 +96,7 @@ func UploadCmd(ctx context.Context, sigRef, payloadRef, imageRef string) error {
 	if err != nil {
 		return err
 	}
-	if _, err := cosign.Upload(ctx, sigBytes, payload, dstRef, authn.DefaultKeychain, cosign.UploadOpts{}); err != nil {
+	if _, err := cosign.Upload(ctx, sigBytes, payload, dstRef, cosign.UploadOpts{}, remote.WithAuthFromKeychain(authn.DefaultKeychain)); err != nil {
```

Comment:
wdyt about putting this actually into UploadOpts as a slice?

Reply:
done